src/main/java/net/minecraftforge/fluids/BlockFluidBase.java

```diff
@@ -492,6 +499,25 @@ public IBlockState getExtendedState(@Nonnull IBlockState oldState, @Nonnull IBlo
                 }
             }
         }
+        
+        //check for downflow above corners
+        if(isCoveredWithFluid(worldIn, pos.add(-1, 0, -1)) || isCoveredWithFluid(worldIn, pos.add(-1, 0, 0)) || isCoveredWithFluid(worldIn, pos.add(0, 0, -1)))
+        {
+            corner[0][0] = 1;
+        }
+        if(isCoveredWithFluid(worldIn, pos.add(-1, 0, 0)) || isCoveredWithFluid(worldIn, pos.add(-1, 0, 1)) || isCoveredWithFluid(worldIn, pos.add(0, 0, 1)))
+        {
+            corner[0][1] = 1;
+        }
+        if(isCoveredWithFluid(worldIn, pos.add(0, 0, -1)) || isCoveredWithFluid(worldIn, pos.add(1, 0, -1)) || isCoveredWithFluid(worldIn, pos.add(1, 0, 0)))
+        {
+            corner[1][0] = 1;
+        }
+        if(isCoveredWithFluid(worldIn, pos.add(1, 0, 0)) || isCoveredWithFluid(worldIn, pos.add(0, 0, 1)) || isCoveredWithFluid(worldIn, pos.add(1, 0, 1)))
+        {
+            corner[1][1] = 1;
+        }
+        
```

Comment:
Hard question. If it was Ender IO, I'd slap a config value in ;)

I'd say this is something for the Forge'ers to decide.

(nice color, btw)